src/views/home-view.js

```diff
@@ -123,17 +123,8 @@ class HomeView extends ViewMixin(ReduxMixin(PolymerElement)) {
      src="../../images/index/hero/hero_image_web_components@2600.png" alt="Banner Image">
 
 <div class="container">
-    <div class="row">
-        <div class="row-item">
-            <h1><b>Up to 98% faster integration</b></h1>
-            <p style="text-align: center">
-                With the new FACT-Finder Web Components
-            </p>
-        </div>
-    </div>
 
-    <hr>
-    <div class="link-box-container">
+    <div class="row link-box-container">
```

Comment:
For the record:
`.row` seems to remove the padding between the tiles. Please find a way to make it look the same as before.